pom.xml

```diff
@@ -32,6 +32,8 @@
     <inceptionYear>2018</inceptionYear>
     <properties>
         <main-class>nl.knaw.dans.easy.deposit.Command</main-class>
+        <jwt-core.version>4.3.0</jwt-core.version>
```

Comment:
I do see commits with 4.3.0 but the project readme and last [release](https://github.com/pauldijou/jwt-scala/releases) are still at 4.2.0. Don't know what the implies.

Reply:
Hmm, mvnrepository.com does list 4.3.0 as the latest version https://mvnrepository.com/artifact/com.pauldijou/jwt-core